### DISCUSSION TITLE
FAQ Discussions Page

### DISCUSSION THREAD

**Discussion by clairmont32** [Security-Onion-Solutions/securityonion#5715] [Ideas] [answered]
In the past several days I've been reviewing and attempting to answer some questions, there seem to be a lot of repeated questions; many answered by linking to the docs. @dougburks  @defensivedepth  @TOoSmOotH what do you think about creating a tile to next to the "Read this first" for some of these kinds of questions?

**Comment by dougburks** [comment]
Hi @clairmont32 ,

Really appreciate you taking the time to help folks out here!

https://github.com/Security-Onion-Solutions/securityonion/discussions/1720 (the first tile on the main Discussions page) already has a link to our [FAQ](https://docs.securityonion.net/en/2.3/faq.html#faq).  Did you have something else in mind?

**Comment by clairmont32** [comment]
Honestly, it feels like it may be "in your face" enough as part of the 3rd headline there. Not everyone will read that tile first (lets also admit it, even the FAQ page) but I'm wondering/curious if there'd be a bit less volume if it was a bit more quickly visible or accessible. It'd also give a quick 1 sentence response for linking to the FAQ page when viable.

Some of the questions over the past week have been related to additional log ingestion with filebeats and logstash so it may be worth adding a FAQ section there too (I could take a stab at that later).

**Comment by dougburks** [comment, marked as answer]
Let's try it and see!

I've created the following discussion and pinned to the top:
https://github.com/Security-Onion-Solutions/securityonion/discussions/5716

**Reply by clairmont32** [reply]
Sounds good! Thanks
